### PR TITLE
Laba 5 

### PR DESCRIPTION
**Pull request by AntonKazlouski**
The results

### CODE REVIEW

File: Anton_Kazlouski/05.Ansible.Run/05.Ansible.run.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+Create playbook ansiblerunuser.yaml in the folder 05.Ansible.Run as follows:
+
+```bash
+
+---
+ - hosts: laba5hosts
+   become: yes 
+   gather_facts: no
+   vars:
+    - user_pass: 'Qwerty1'
```
Comment (pluhin) [inline Anton_Kazlouski/05.Ansible.Run/05.Ansible.run.md]: Why is that doing here, ansible vault!!!

Comment (pluhin) [reply Anton_Kazlouski/05.Ansible.Run/05.Ansible.run.md]: Or should be provided like extra var

File: Anton_Kazlouski/05.Ansible.Run/ansiblerunuser.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+ - hosts: laba5hosts
+   become: yes 
+   gather_facts: no
+   vars:
+    - user_pass: 'Qwerty1'
```
Comment (pluhin) [inline Anton_Kazlouski/05.Ansible.Run/ansiblerunuser.yaml]: put into vault or extravars

File: Anton_Kazlouski/05.Ansible.Run/05.Ansible.run.md (hunk 1 above)
Comment (AntonKazlouski) [reply Anton_Kazlouski/05.Ansible.Run/05.Ansible.run.md]: Fixed

File: Anton_Kazlouski/05.Ansible.Run/ansiblerunuser.yaml (hunk 2 above)
Comment (AntonKazlouski) [reply Anton_Kazlouski/05.Ansible.Run/ansiblerunuser.yaml]: Fixed

**Comment by AntonKazlouski** [comment]
The problem has been successfully fixed

**Pull request merged**